core/modules/utils/performance.js

```diff
@@ -68,14 +76,14 @@ Performance.prototype.log = function() {
 /*
 Wrap performance measurements around a subfunction
 */
-Performance.prototype.measure = function(name,fn) {
+Performance.prototype.measureFn = function(name, description,fn) {
 	var self = this;
 	if(this.enabled) {
-		return function() {
+		return function measureCallback() {
```

Comment:
Add a name to the function so it has a name when debugging, this is a best practice from eslint.